src/components/levels.js

```diff
@@ -1,15 +1,45 @@
 import * as stickers from './Stickers.js';
 import { childPos, childSize } from './Stickers.js';
-import React from 'react';
-
-class Hex extends React.Component{
-    render(){
-        return (
-            <span>
-                <text onClick={() => {navigator.clipboard.writeText(this.props.hexcode)}} style={{color:this.props.hexcode}}>({this.props.hexcode})</text>
-            </span>
-        );
+import React, { useState, useEffect } from 'react';
+
+function Hex(props){
+    const [text, setText] = useState(props.hexcode);
```

Comment:
To prevent human error of text being something other than `Copied!` because it's equality is tested in a different place than it being used (L10 & L35), in addition to only having 2 states, we should make this a boolean such as `isCopied` or `isShowTextCopied`
```suggestion
    const [i,sCopied setIsCopied] = useState(false);
```